[ShiftStack: OSDOCS-1877] Add provider network installation docs (#32632)
openshift/openshift-docs: modules/installation-osp-deploying-provider-networks-installer.adoc

// Module included in the following assemblies:
//
// * list of assemblies where this module is included

> // TODO: Add mods to all appropriate installation assemblies after first pass is approved.

Reviewer: Just a reminder on these TODOs.

Author: Fixed in https://github.com/openshift/openshift-docs/pull/32632/commits/6d8c0f8381a479ae136618c130f7bfbace23e9a5

[id="installation-osp-deploying-provider-networks-installer_{context}"]
= Deploying a cluster that has a primary interface on a provider network

> 

Author: ```suggestion

You can deploy an {product-title} cluster that has its primary network interface on an {rh-openstack-first} provider network.
```

> .Prerequisites

Reviewer: Maybe add a short procedure intro before the prereqs.

* Your {rh-openstack-first} deployment is configured as described by "{rh-openstack} provider network requirements for cluster installation".

.Procedure

. In a text editor, open the `install-config.yaml` file.
. Set the value of the `platform.openstack.apiVIP` property to the IP address for the API VIP.
. Set the value of the `platform.openstack.ingressVIP` property to the IP address for the Ingress VIP.
. Set the value of the `platform.openstack.machinesSubnet` property to the UUID of the provider network subnet.
. Set the value of the `networking.machineNetwork.cidr` property to the CIDR block of the provider network subnet.

[IMPORTANT]
====

> The `platform.openstack.apiVIP` and `platform.openstack.ingressVIP` properties both need to be unassigned IP addresses from the `networking.machineNetwork.cidr` block.

Reviewer: s/need to/must/

Author: ```suggestion
The `platform.openstack.apiVIP` and `platform.openstack.ingressVIP` properties must both be unassigned IP addresses from the `networking.machineNetwork.cidr` block.
```
